pkgs/by-name/q2/q2pro/package.nix

```diff
@@ -19,32 +19,39 @@
   libdecor,
   ffmpeg,
   wayland-scanner,
-  makeWrapper,
+  makeBinaryWrapper,
   versionCheckHook,
+  copyDesktopItems,
+  makeDesktopItem,
+  desktopToDarwinBundle,
   x11Support ? stdenv.hostPlatform.isLinux,
   waylandSupport ? stdenv.hostPlatform.isLinux,
 }:
 
 stdenv.mkDerivation (finalAttrs: rec {
   pname = "q2pro";
-  version = "3510";
```

Comment:
The r{xxxx} tags are not more significant than any commits in master and shouldn't be considered as a form of releases - https://github.com/skullernet/q2pro/issues/363#issuecomment-2442448224